specification/subscription/resource-manager/readme.md

```diff
@@ -44,6 +44,13 @@ directive:
   - suppress: DeleteOperationResponses
     from: subscriptions.json
     reason: The delete operation response for subscription changed directory expected to return 200 on every user's request, once it's deleted it will return 404, since it's a change on the existing api with new version, but creating workitem - "https://msazure.visualstudio.com/One/_workitems/edit/29188912", to refactor the call on delete request and will return 204 as no content in such cases to take this item in future ref.
+  - suppress: PutResponseCodes
+    from: subscriptions.json
+    where: $.paths["/providers/Microsoft.Subscription/transfers/{transferId}"].put
+    reason: Not supposed to return 201 as the response code for the below API since existing api with new version change, got exceptions from ARM reviewer.
```

Comment:
>Not supposed to return 201 as the response code for the below API since existing api with new version change, got exceptions from ARM reviewer. [](http://example.com/codeflow?start=12&length=143)

Could you please provide where you got the exception? If this is async PUT, 200 alone will not suffice per RPC. Linked RPC at the API implementation.